sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -821,7 +821,7 @@ public async Task StartRecognizeReceiptsCanParseMultipageForm(bool useStream)
                     var sampleField = recognizedForm.Fields["MerchantAddress"];
 
                     Assert.IsNotNull(sampleField.ValueData);
-                    Assert.AreEqual("2345 Dogwood Lane Birch, Kansas 98123", sampleField.ValueData.Text);
+                    Assert.AreEqual("Maple City, Massachusetts.", sampleField.ValueData.Text);
```

Comment:
This one is a regression, but something already pointed out by the service team is that we're using the receipts prebuilt model to recognize multipage invoices, so it's only fair that it won't work as intended. I think we should get proper files for these tests.

Reply:
Agree, we should change them

Reply:
Could you create an issue for that?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/19826